mrq/dashboard/app.py

```diff
@@ -54,7 +54,7 @@ def root():
 @requires_auth
 def api_task_exceptions():
     stats = list(connections.mongodb_jobs.mrq_jobs.aggregate([
-        {"$match": {"status": "failed"}},
+        {"$match": {"status": ["failed","maxretries"]}},
```

Comment:
`{"$match": {"status": {"$in": ["failed","maxretries"]}}},`

Reply:
This is the right way, Thank you.